specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/computeRPCommon.json

```diff
@@ -1113,6 +1113,14 @@
         "configurationReference": {
           "type": "string",
           "description": "Optional, Specifies the uri to an azure blob that will replace the default configuration for the package if provided"
+        },
+        "treatFailureAsDeploymentFailure": {
```

Comment:
these properties are being added, but not being used in by any resources in 2022-03-01. 
Also, only if they are being used across ComputeRP Resources, add it to computeRPCommon.json. 
Otherwise, please add to resource specific. 
I am unable to tell what level they should be in because there are no references to these properties